src/txn/txn.c

```diff
@@ -751,6 +755,21 @@ __txn_commit_timestamps_assert(WT_SESSION_IMPL *session)
 			 */
 			if (upd == NULL)
```

Comment:
@agorrod @bvpvamsikrishna 
I think we need to close the cursor here too.

Reply:
c464c44

Reply:
I agree. I'd prefer to wrap the assignments at line 759 and 760 in an `if (upd != NULL)`, and then move the continue below the cursor close.

Reply:
Sounds good. If I'm understanding your comment, I think that's what I did in the commit above.